api/village.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+# Begin standard auth
+require_once __DIR__ . "/../classes.php";
+
+$system = API::init(row_lock: true);
+
+try {
+    $player = Auth::getUserFromSession($system);
+    $player->loadData(User::UPDATE_NOTHING);
+} catch(RuntimeException $e) {
+    API::exitWithException($e, system: $system);
+}
+
+try {
+    // api requires a request
+    if (isset($_POST['request'])) {
+        $request = filter_input(INPUT_POST, 'request', FILTER_SANITIZE_FULL_SPECIAL_CHARS);
+    } else {
+        throw new RuntimeException('No request was made!');
+    }
+
+    $VillageAPIResponse = new VillageAPIResponse();
+
+    switch($request) {
+        case 'LoadVillageData':
+            $VillageAPIResponse->response = [
+                'policyData' => VillageApiPresenter::boostDataResponse($system, $player),
```

Comment:
Chi is still working on boosts/policies so this is unused for now

Reply:
Should we comment out this line or add a placeholder function returning an empty array then? Seems like this should throw an error since we're trying to call a function that doesn't exist